quickstart-client/src/main/java/org/wildfly/extras/quickstart/microprofile/graphql/client/FilmResource.java

```diff
@@ -0,0 +1,109 @@
+package org.wildfly.extras.quickstart.microprofile.graphql.client;
+
+import io.smallrye.graphql.client.Response;
+import io.smallrye.graphql.client.core.Document;
+import io.smallrye.graphql.client.core.ScalarType;
+import io.smallrye.graphql.client.core.Variable;
+import io.smallrye.graphql.client.dynamic.api.DynamicGraphQLClient;
+import io.smallrye.graphql.client.dynamic.api.DynamicGraphQLClientBuilder;
+import io.smallrye.graphql.client.typesafe.api.TypesafeGraphQLClientBuilder;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import org.wildfly.extras.quickstart.microprofile.graphql.client.model.Film;
+import org.wildfly.extras.quickstart.microprofile.graphql.client.model.Hero;
+
+import java.util.Collections;
+import java.util.List;
+
+import static io.smallrye.graphql.client.core.Argument.arg;
+import static io.smallrye.graphql.client.core.Argument.args;
+import static io.smallrye.graphql.client.core.Document.document;
+import static io.smallrye.graphql.client.core.Field.field;
+import static io.smallrye.graphql.client.core.Operation.operation;
+import static io.smallrye.graphql.client.core.Variable.var;
+import static io.smallrye.graphql.client.core.Variable.vars;
+import static io.smallrye.graphql.client.core.VariableType.nonNull;
+
+@Path("/")
+public class FilmResource {
+
+    private static final String url;
+
+    static {
+        final Integer port = Integer.getInteger("port", 8080);
+        url = "http://localhost:" + port + "/quickstart/graphql";
+    }
+
+    @GET
+    @Path("/typesafe/films")
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<Film> getAllFilms() {
+        System.out.println("Using URL " + url);
+        FilmClientApi client = TypesafeGraphQLClientBuilder.newBuilder()
```

Comment:
I think we could inject `ClientModels` in this class and pass it to the builders